### ISSUE TITLE
Error when using this module in atom editor

### ISSUE THREAD

**Issue by abtevrythng** [tcort/link-check#6]
[Steps to reproduce:]

1. Generate new package with package generator [From guide here](http://flight-manual.atom.io/hacking-atom/sections/package-word-count/)
2. require link-check module
3. Use it somewhere in code

**Atom**: 1.17.2 x64
**Electron**: 1.3.15
**OS**: "Fedora release
**Thrown From**: test-linkcheck package 1.0.0

### Stack Trace

Failed to activate the test-linkcheck package

```
At Identifier 'process' has already been declared

SyntaxError: Identifier 'process' has already been declared
    at Module._compile (/usr/share/atom/resources/app/src/native-compile-cache.js:75:40)
    at Object.value [as .js] (/usr/share/atom/resources/app/src/compile-cache.js:225:23)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (/usr/share/atom/resources/app/src/native-compile-cache.js:47:27)
    at /packages/atom-test-linkcheck/node_modules/link-check/index.js:5:11)
    at Module._compile (/usr/share/atom/resources/app/src/native-compile-cache.js:87:30)
    at Object.value [as .js] (/usr/share/atom/resources/app/src/compile-cache.js:225:23)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (/usr/share/atom/resources/app/src/native-compile-cache.js:47:27)
    at /packages/atom-test-linkcheck/lib/test-linkcheck.js:15:17)
    at Module._compile (/usr/share/atom/resources/app/src/native-compile-cache.js:87:30)
    at Object.value [as .js] (/usr/share/atom/resources/app/src/compile-cache.js:225:23)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (/usr/share/atom/resources/app/src/native-compile-cache.js:47:27)
    at Package.module.exports.Package.requireMainModule (/usr/share/atom/resources/app/src/package.js:885:29)
    at Package.module.exports.Package.activateNow (/usr/share/atom/resources/app/src/package.js:240:16)
    at /usr/share/atom/resources/app/src/package.js:223:32
    at Package.module.exports.Package.measure (/usr/share/atom/resources/app/src/package.js:97:15)
    at /usr/share/atom/resources/app/src/package.js:216:26
    at Package.module.exports.Package.activate (/usr/share/atom/resources/app/src/package.js:213:34)
    at PackageManager.module.exports.PackageManager.activatePackage (/usr/share/atom/resources/app/src/package-manager.js:640:34)
```

### Commands

```
     -0:42 window:toggle-dev-tools (a.stack-toggle)
```

**Comment by gaurav-nelson** [comment]
Fixes in [PR#7:fix for interoperatibility with atom and electron based applications ](https://github.com/tcort/link-check/pull/7)

**Comment by tcort** [comment]
Thanks for the well written bug report (and the patch!). PR#7 has been merged, and I've published a new version with the fix (version `4.0.3`
